Bearded.UI/Controls/implementations/Button.cs

```diff
@@ -21,11 +22,26 @@ public override void MouseButtonReleased(MouseButtonEventArgs eventArgs)
             base.MouseButtonReleased(eventArgs);
             if (eventArgs.MouseButton == MouseButton.Left && IsEnabled)
             {
-                Clicked?.Invoke();
+                FireClickEvent(new ClickEventArgs(eventArgs.ModifierKeys));
             }
             eventArgs.Handled = true;
         }
 
+        protected void FireClickEvent(ClickEventArgs eventArgs)
```

Comment:
Why isn't this just Click() (and possibly public?)?

Reply:
Updated, but not sure if I like it. Feels a bit like `Click` would invoke the normal mouse click events, which it doesn't, and that `Trigger` would be more obvious to a caller. But looking at the events of `Control`, the only connection to "click" is the `IsClickThrough` property so it's probably fine.

Reply:
Good catch though, 'is click through' does deal with cursor clicks specifically, right? It wouldn't apply to  keyboard navigation. Oh... taps as well... Naming is hard :(